book.rb

```diff
@@ -0,0 +1,16 @@
+# book class
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] Getters and Setters for @rentals should be present to create the many-to-many relationship between the class Person and Book. Let's fix that

Reply:
I'll fix it @EricMbouwe 👍🏼